pandas/tseries/tools.py

```diff
@@ -307,10 +307,19 @@ def _to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     """
     Same as to_datetime, but accept freq for
     DatetimeIndex internal construction
+
+    Parameters
+    ----------
+    freq : string
```

Comment:
@jreback good catch. The parameter looks no longer used from ``to_datetime``, thus can be removed.